src/register.ts

```diff
@@ -56,8 +57,10 @@ async function registerGuildCommands(
   }
 }
 
+const envLoadedFromProcess = loadEnv({}, false);
+
 registerGuildCommands(
-  process.env.DISCORD_TOKEN,
-  process.env.DISCORD_APPLICATION_ID,
-  process.env.DISCORD_GUILD_ID
+  envLoadedFromProcess.DISCORD_TOKEN,
+  envLoadedFromProcess.DISCORD_APPLICATION_ID,
+  envLoadedFromProcess.DISCORD_GUILD_ID
```

Comment:
Please rename the envLoadedFromProcess?

Reply:
is `envVariablesFromProcess` ok ?
or `envFromProcessEnv`

Reply:
Is it just an env key? why so long name?

Reply:
It contains all the env keys loaded from through process.env, in index.ts we access the env injected into the worker not from process.env [link](https://github.com/Saitharun279/discord-slash-commands/blob/e29109b7fd5270e12741bd48b93cc2f6f8833c5e/src/index.ts#L110). 
Here, in register.ts we access env from process.env, so to clearly differentiate this from the env in index.ts, i named it envLoadedFromProcess.

Reply:
Also it will be easier to understand by reading the name, otherwise we need to look into the function call where env object is being created.

Reply:
I feel its a long name, what is the use of telling it's loaded from the process or something if we don't have multiple, and if this is the default that needs to be used always?

Reply:
ok, should i change it and use just `env` ?

Reply:
See, what ever fits the best

Reply:
Actually we do access env vars in 2 different ways, 

1. from worker while the application is live, the [worker secrets](https://developers.cloudflare.com/workers/configuration/secrets/) are provided as env object to the [fetch handler](https://github.com/Saitharun279/discord-slash-commands/blob/e29109b7fd5270e12741bd48b93cc2f6f8833c5e/src/index.ts#L97) or [scheduled handler](https://github.com/Saitharun279/discord-slash-commands/blob/e29109b7fd5270e12741bd48b93cc2f6f8833c5e/src/index.ts#L114). 
2. And from process.env in CD pipeline in Github Actions, where env var validation will happen

Since these env in [register.ts](https://github.com/Saitharun279/discord-slash-commands/blob/e29109b7fd5270e12741bd48b93cc2f6f8833c5e/.github/workflows/register-commands-staging.yaml#L39) are accessed only during the CD pipeline, i named it `envLoadedFromProcess`

Reply:
when the code runs it needs env only, in run time why will i care about the CI?

Reply:
Yes, I get what you are saying.
That is the reason why I was asking to get a third-person view. Since we both are in a disagreement here